src/test/scala/fte/data/GraphCreator.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ Copyright 2016 Jo Pol
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see http://www.gnu.org/licenses/gpl.html dibl
+*/
+package fte.data
+
+import dibl.proto.TilesConfig
+import dibl.{ LinkProps, NewPairDiagram }
+import fte.layout.OneFormTorus
+
+object GraphCreator {
+
+  /**
+    * @param urlQuery parameters for: https://d-bl.github.io/GroundForge/tiles?
+    *                 The patch size must span 3 columns and 2 rows of checker tiles.
+    *                 In ascii-art:
+    *                 +----+----+----+
+    *                 |....|....|....|
+    *                 |....|....|....|
+    *                 +----+----+----+
+    *                 |....|....|....|
+    *                 |....|....|....|
+    *                 +----+----+----+
+    * @return
+    */
+  def fromPairs(urlQuery: String): Option[Graph] = {
```

Comment:
The method is called with the following strings
https://github.com/d-bl/GroundForge/blob/cea01f261e0875227c2e975b5d9875b7740bf10b/src/test/scala/fte/Demo.scala#L26-L28
You can add these strings to the link https://d-bl.github.io/GroundForge/tiles? to experiment with the patterns.